### DISCUSSION TITLE
Resource alias

### DISCUSSION THREAD

**Discussion by rabbitism** [AvaloniaUI/Avalonia#8773] [Q&A] [answered]
Hello,
Is it possible to add a mechanism to define resource to be just an alias of another resource? This is widely used as design tokens in CSS frameworks. 
```css
$font-typography_title-fontWeight: $font-weight-bold; 
$font-typography_link-fontWeight: $font-weight-bold; 
$font-typography_strong-fontWeight: $font-weight-bold; 
```

**Comment by maxkatz6** [comment, marked as answer]
Yes, you can:
`<StaticResource x:Key="NewKey" ResourceKey="OriginalKey" />`

**Reply by rabbitism** [reply]
Thanks! that's what I'm looking for!
